packtools/sps/validation/aff.py

```diff
@@ -411,9 +401,9 @@ def validate_comparison(self, main_aff: dict):
             got = result["got"]
             items = result["items"]
             if self.affiliation.get("id"):
-                advice = f"Review affiliation ({self.affiliation.get('id')})"
+                advice = f"Low similarity detected. Review and adjust affiliation ({self.affiliation.get('id')}) to better match the main text."
```

Comment:
@Rossi-Luciano Use um texto menos técnico.

`The affiliation {self.affiliation} is inconsistent with {main_aff}. Check if {self.affiliation} is translation of {main_aff}`

Reply:
OK, @robertatakenaka